### PR TITLE
handle progressbar having 0 seconds left at startup

### PR DESCRIPTION
**Pull request by defhacks**
Attempt to address https://github.com/gopasspw/gopass/issues/2046

I'm not sure if the release notes commit message instructions are for my commit, or for the merge commit, but if I need change thing, I'm than happy to.

### CODE REVIEW

File: pkg/termio/progress.go [hunk 1]
```diff
@@ -129,7 +129,14 @@ func (p *ProgressBar) doPrint() {
 	}
 
 	barWidth := uint(termWidth)
-	digits := int(math.Log10(float64(max))) + 1
+	var digits int
+	// Log10(0) is undefined
+	if max > 0 {
```
Comment (dominikschulz) [inline pkg/termio/progress.go:134]: I think we can make this a little bit shorter.

Could you leave the initial assignment as is and only add three lines to check max < 1 and then set the value to 1.

**Comment by AnomalRoil** [comment]
For your commit to pass CI, you need to sign it off using the syntax 
`git commit -s` plus what you usually do.

And also to add a Release note message, so the easy way is to do :
`git commit - - amend -s -m "RELEASE_NOTES=[BUGFIX] Fixing progress bar to handle zero properly"`
And then `git push --force` to correct your branch.

**Comment by defhacks** [comment]
Hopefully that's all correct now, it probably ended up being a bit more work having me do it, but I do appreciate the guidance and also I learned about `git commit -s`  :)

**Review by dominikschulz — APPROVED** [review]
LGTM

**Pull request merged**